apps/src/code-studio/pd/workshop_dashboard/components/move_enrollments_dialog.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Modal, Button} from 'react-bootstrap';
+
+const styles = {
+  warning: {
+    color: 'red'
+  }
+};
+
+export default class MoveEnrollmentsDialog extends React.Component {
+  static propTypes = {
+    selectedEnrollments: PropTypes.array,
+    show: PropTypes.bool,
+    onCancel: PropTypes.func,
+    onMove: PropTypes.func
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      destination_workshop_id: null
+    };
+
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.handleClickMove = this.handleClickMove.bind(this);
+  }
+
+  handleInputChange(e) {
+    this.setState({destination_workshop_id: e.target.value});
+  }
+
+  handleClickMove() {
+    this.props.onMove(this.state.destination_workshop_id);
+  }
+
+  renderSelectedEnrollments() {
+    const enrollments = this.props.selectedEnrollments.map((enrollment, i) => {
+      return (
+        <li key={i}>
+          {enrollment.first_name} {enrollment.last_name} ({enrollment.email})
+        </li>
+      );
+    });
+
+    return <ul>{enrollments}</ul>;
+  }
```

Comment:
Style nit: I'd prefer to see this as a separate stateless component, rather than a helper method.

```jsx
// Functional stateless component in the same file
const EnrollmentList = ({enrollments}) => (
  <ul>
    {enrollments.map((e, i) => (
      <li key={i}>
        {e.first_name} {e.last_name} ({e.email})
      </li>
    )}
  </ul>
);

// Used in the render method later:
- {this.renderSelectedEnrollments()}
+ <EnrollmentList enrollments={this.props.selectedEnrollments}/>
```